packages/api/src/commands/list.ts

```diff
@@ -0,0 +1,42 @@
+import chalk from 'chalk';
+import { Command } from 'commander';
+
+import logger from '../logger.js';
+import Storage from '../storage.js';
+
+const cmd = new Command();
+cmd
+  .name('list')
```

Comment:
When thinking about this command last year I originally had plans to support `npx api list petstore` but don't really think that's worth it anymore. This list of installed APIs isn't ever going to be the size of something like `npm ls` so filtering isn't something I can ever see anybody ever want.